### DISCUSSION TITLE
Unable to push docker image to GPR: manifest invalid

### DISCUSSION THREAD

**Discussion by pavel-spacil** [community/community#25499] [Actions]
<p>Hello, starting today, I’m receiving the error “manifest invalid: Only schema version 2 is supported” when pushing docker image to my repository. It’s failing the same in Actions and locally when trying to push images. No change on the actions or my local settings. The last push 15 days ago succeed so not sure what happened in the meantime. From the org perspective there is enough space and bandwidth available in the billing section so this might not be the issue (as pointed by someone elsewhere). Any clues what to look on? Docker debug logs say nothing helpful, running on Win with Linux containers.</p>
<p>Thanks<br>
Pavel</p>

**Comment by markphelps** [comment]
<p>Hi <a class="mention" href="/pavel-spacil">@pavel-spacil</a>, GHCR engineer here. Could you please include the name of the docker image (full path like org/image-name) please so we can help debug?</p>

**Comment by pavel-spacil** [comment]
<p>Hi Mark, the image is <code>solarwinds/marble</code>, I’ve tried to push various tags there. Thanks</p>

**Comment by markphelps** [comment, marked as answer]
<p>Hi <a class="mention" href="/pavel-spacil">@pavel-spacil</a>, I see the token you are using has the <code>write:packages</code> scope, could you do me a favor and try using a new token that has <code>write:packages, read:packages and repo</code> scopes?</p>

**Comment by pavel-spacil** [comment]
<p>Hi <a class="mention" href="/markphelps">@markphelps</a>, it seems that even just <code>repo_deployment</code> scope did the trick and I’m able to push images locally and on CI again! Thank you</p>
